sdk/event/event.go

```diff
@@ -31,25 +29,23 @@ func New(ps *pubsub.PubSub, service *servicesdk.Service, instance *instancesdk.I
 }
 
 // Emit emits a MESG event eventKey with eventData for service token.
-func (e *Event) Emit(instanceHash hash.Hash, eventKey string, eventData map[string]interface{}) (*event.Event, error) {
-	instance, err := e.instance.Get(instanceHash)
+func (e *Event) Create(event *event.Event) error {
+	instance, err := e.instance.Get(event.InstanceHash)
```

Comment:
I don't really like this pattern where we send an initialized data to create this data... 
Any create should have the following signature `Create(inputs WhateverStructOrListOfInputs) (ResourceType, error)`